README.md

```diff
@@ -124,6 +124,57 @@ Using JSON payload for constructing a message is also available:
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+### Update the message
+
+Using `update-ts: ${{ steps.slack.outputs.ts }}` will update the message that was posted. 
+
+```yaml
+- id: slack
+  uses: slackapi/slack-github-action@v1.18.0
```

Comment:
The current version is 1.19.0. The feature can be supported in the next version or later.
```suggestion
  uses: slackapi/slack-github-action@v1.20.0
```